packages/core/src/components/button/button.css

```diff
@@ -50,10 +50,20 @@
   outline: none;
 }
 
+.hds-button:active {
+  background-color: var(--background-color-focus, transparent);
+  color: var(--color-focus);
+  outline: none;
+}
+
 .hds-button:focus:hover {
   background-color: var(--background-color-hover-focus, transparent);
 }
 
+.hds-button:active:hover {
+  background-color: var(--background-color-hover-focus, transparent);
```

Comment:
Could the active state be different from the hover state? Then the user could see the click or touch